bin/reth/src/commands/import.rs

```diff
@@ -68,9 +68,9 @@ pub struct ImportCommand {
     )]
     chain: Arc<ChainSpec>,
 
-    /// Disables execution stage.
+    /// Disables stages that require state.
     #[arg(long, verbatim_doc_comment)]
-    disable_execution: bool,
+    no_state: bool,
```

Comment:
are we ok with dropping `disable_execution`? should verify we don't use this in eg hive

Reply:
I introduced it in a recent pr, so we're good

Reply:
maybe `disable_execution` is better more standard terminology though. for me I was associating execution with execution stage, but starting to realise execution is kind of generally associated with state. wdyt?